### PR TITLE
Feature/additional tableview demo

### PR DESCRIPTION
**Pull request by andykkt**

### CODE REVIEW

File: BFWControls Demo/Modules/Table/Controller/DynamicCellDemoViewController.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    fileprivate var rowCount = 0
+
+    // MARK: - Private functions
+    fileprivate func insertCell() {
+        rowCount += 1
+        tableView.insertRows(at: [IndexPath(row: rowCount-1, section: 0)], with: .right)
+        reloadAllVisibleRows()
+    }
+    
+    fileprivate func removeCell(indexPath: IndexPath) {
```
Comment (barefeettom) [inline BFWControls Demo/Modules/Table/Controller/DynamicCellDemoViewController.swift]: For consistency, can we name this `deleteRow(at:)`?

File: BFWControls Demo/Modules/Table/Controller/DynamicCellDemoViewController.swift [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "dynamic", for: indexPath)
+        cell.textLabel?.text = "rowIndex: \(indexPath.row)"
+        return cell
+    }
+    
+    //MARK: - UITableViewDelegate
+    override  func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
```
Comment (barefeettom) [inline BFWControls Demo/Modules/Table/Controller/DynamicCellDemoViewController.swift]: Remove double space.

File: BFWControls Demo/Modules/Table/Controller/DynamicCellDemoViewController.swift [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+class DynamicCellDemoViewController: StaticTableViewController {
+
+    // MARK: - Variables
+    fileprivate var rowCount = 0
+
+    // MARK: - Private functions
+    fileprivate func insertCell() {
+        rowCount += 1
+        tableView.insertRows(at: [IndexPath(row: rowCount-1, section: 0)], with: .right)
```
Comment (barefeettom) [inline BFWControls Demo/Modules/Table/Controller/DynamicCellDemoViewController.swift]: Add spaces around `-` operator.

**Review by barefeettom — CHANGES_REQUESTED** [review]

File: BFWControls Demo/Modules/Table/Controller/DynamicCellDemoViewController.swift (hunk 3 above)
Comment (andykkt) [reply BFWControls Demo/Modules/Table/Controller/DynamicCellDemoViewController.swift]: Added, thanks

File: BFWControls Demo/Modules/Table/Controller/DynamicCellDemoViewController.swift (hunk 1 above)
Comment (andykkt) [reply BFWControls Demo/Modules/Table/Controller/DynamicCellDemoViewController.swift]: Changed function to `deleteRow(at indexPath:)`

File: BFWControls Demo/Modules/Table/Controller/DynamicCellDemoViewController.swift (hunk 2 above)
Comment (andykkt) [reply BFWControls Demo/Modules/Table/Controller/DynamicCellDemoViewController.swift]: Removed, thanks

**Pull request merged**